test/test_runner.go

```diff
@@ -225,13 +232,21 @@ func recoverPanics(onError func(error)) {
 }
 
 func (r *TestRunner) parseCheckAndInterpret(script string) (*interpreter.Program, *interpreter.Interpreter, error) {
-	env := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
+	config := runtime.Config{}
+	if r.coverageReport != nil {
+		config.CoverageReportingEnabled = true
+	}
```

Comment:
```suggestion
	config := runtime.Config{
		CoverageReportingEnabled: r.coverageReport != nil,
	}
```

Reply:
Good catch :+1: 
Fixed in https://github.com/onflow/cadence-tools/pull/78/commits/408b2904c6eb5cc287389c797121d0160c2af0c6